pkg/bindings/test/pods_test.go

```diff
@@ -190,7 +190,45 @@ var _ = Describe("Podman images", func() {
 		}
 	})
 
-	// Remove all stopped pods and their container to be implemented.
+	// Test to validate all the pods in the exited state are pruned sucessfully.
 	It("prune pod", func() {
+		// Add a new pod
+		var newpod2 string = "newpod2"
+		bt.Podcreate(&newpod2)
+		// No pods pruned since no pod in exited state
+		err = pods.Prune(connText)
+		Expect(err).To(BeNil())
+		podSummary, err := pods.List(connText, nil)
+		Expect(err).To(BeNil())
+		Expect(len(podSummary)).To(Equal(2))
+
+		// Prune only one pod which is in exited state.
+		// Start then stop a pod.
+		// pod moves to exited state one pod should be pruned now.
+		err = pods.Start(connText, newpod)
+		Expect(err).To(BeNil())
+		err = pods.Stop(connText, newpod, nil)
+		Expect(err).To(BeNil())
+		err = pods.Prune(connText)
+		Expect(err).To(BeNil())
+		podSummary, err = pods.List(connText, nil)
+		Expect(err).To(BeNil())
+		Expect(len(podSummary)).To(Equal(1))
+
+		// Test prune all pods in exited state.
+		bt.Podcreate(&newpod)
+		err = pods.Start(connText, newpod)
+		Expect(err).To(BeNil())
+		err = pods.Start(connText, newpod2)
+		Expect(err).To(BeNil())
+		err = pods.Stop(connText, newpod, nil)
+		Expect(err).To(BeNil())
+		err = pods.Stop(connText, newpod2, nil)
+		Expect(err).To(BeNil())
```

Comment:
it might be good to add a test somewhere in here to make sure you've two pods up and in the exited state before you remove them.

Reply:
Currently, the pod inspect doesn't return the pod status I am working on updating the pod inspect bindings and will do it in the following pr.

Reply:
fixed